src/pages/controller.html

```diff
@@ -231,7 +231,7 @@ <h2>Question control area</h2>
                                                 </button>
                                                 <button type="button" class="secondary"
                                                     x-bind:disabled="currentQuestion.closed"
-                                                    x-on:click="currentQuestion.fails.teamA.increase()">
+                                                    x-on:click="if (currentQuestion.fails.teamA.failCount != 3) { currentQuestion.fails.teamA.increase() }">
```

Comment:
In increase und decrease ist eigentlich schon ein Guard drin, der dafür sorgt, dass der Wert zwischen 0 und 3 bleibt, siehe https://github.com/MaxMuehlbauer/familienduell/blob/main/src/scripts/gameState.ts#L43